Sources/web3swift/Tokens/ERC1376/Web3+ERC1376.swift

```diff
@@ -66,20 +66,15 @@ protocol IERC1376: IERC20 {
 }
 
 // FIXME: Rewrite this to CodableTransaction
-public class ERC1376: IERC1376, ERC20BaseProperties {
-
-    internal var _name: String?
-    internal var _symbol: String?
-    internal var _decimals: UInt8?
-    internal var _hasReadProperties: Bool = false
-
+public class ERC1376: IERC1376, ERCBaseProperties {
+    public var basePropertiesProvder: ERCBasePropertiesProvider
```

Comment:
Additionally, I'd make it `public private(set) var` and the same goes to the `public lazy var contract: Web3.Contract` on line 77.

If a developer wants a new `ERC1376` I don't think it would be easier to change every single attribute in an instance of this class. I'd call `ERC1376.ini(...)` again instead to be sure the data is set correctly.